src/Select/web/Select.js

```diff
@@ -172,40 +178,46 @@ class Select extends React.Component {
                   <OptionList
                     isOpen={isOpen}
                     block={block}
+                    height={optionsListHeight}
                   >
-                    {
-                      options
-                        .map(this.makeOption)
-                        .map((option, index) => (
-                          <Option
-                            {...getItemProps({
-                              key: option.value,
+                    <VirtualList
+                      width="100%"
+                      height={Number.parseInt(theme.px(optionsListHeight), 10)}
```

Comment:
Once you remove max-height(as mentioned above), you can now directly pass `27 * 8` here

Reply:
Instead of directly passing `27 * 8` can we do something in a better way? Like we use the `px` function so that we don't explicitly want to pass scale i.e `8` for now for us. Let's say if someone externally uses this they might have scale defined as let's say `10` now they have to explicitly come here and change this `8` to `10` as well. thoughts?